packages/app-store/dailyvideo/test/video-api-adapter.test.ts

```diff
@@ -0,0 +1,252 @@
+  fetchMocker.mockOnceIf(`${dailyBaseUrl}/meeting-tokens`, JSON.stringify(meetingTokenResponse));
+}
+
+function assertRequestsWithoutPlan(mainFetchBody: object, tokenFetchBody: object) {
+  expect(mainFetchBody).toEqual(expectedBodyMainFetchDefault);
+  expect(tokenFetchBody).toEqual(expectedBodyTokenFetch);
+}
+
+function assertRequestsWithPlan(mainFetchBody: object, tokenFetchBody: object) {
+  expect(mainFetchBody).toEqual(expectedBodyMainFetchWithPlan);
+  expect(tokenFetchBody).toEqual(expectedBodyTokenFetch);
+}
+
+async function assertRequests(withPlan: boolean) {
+  const requests = fetchMocker.requests();
+  const mainRequest = await requests[0].json();
+  const tokenRequest = await requests[1].json();
+  if (withPlan) {
+    assertRequestsWithPlan(mainRequest, tokenRequest);
+  } else {
+    assertRequestsWithoutPlan(mainRequest, tokenRequest);
+  }
+}
+
+async function testCreateMeeting(scalePlan: boolean, teamPlan: boolean) {
+  await commonSetup({ scalePlan, teamPlan, mainFetchEndpoint: "/rooms" });
+  const result = await DailyVideoApiAdapter()?.createMeeting(event);
+  await assertRequests(scalePlan && teamPlan);
+  expect(result).toStrictEqual(createAndUpdateResult);
+}
+
+async function testUpdateMeeting(scalePlan: boolean, teamPlan: boolean) {
+  await commonSetup({ scalePlan, teamPlan, mainFetchEndpoint: `/rooms/${event.uid}` });
+  const result = await DailyVideoApiAdapter()?.updateMeeting(event, event);
+  await assertRequests(scalePlan && teamPlan);
+  expect(result).toStrictEqual(createAndUpdateResult);
+}
+
+const event = {
+  uid: "MOCK_ID",
+  endTime: new Date().getTime(),
+  organizer: {
+    id: "mockid",
+  },
+};
+
+describe("Daily Video API Adapter", () => {
+  describe("Create and update meeting", () => {
+    describe("createMeeting", () => {
+      beforeEach(() => {
+        fetchMocker.mockClear();
+      });
+
+      test("should create meeting with scalePlan: true and teamPlan: true", async () => {
+        await testCreateMeeting(true, true);
+      });
+
+      test("should create meeting with scalePlan: true and teamPlan: false", async () => {
+        await testCreateMeeting(true, false);
+      });
+
+      test("should create meeting with scalePlan: false and teamPlan: true", async () => {
+        await testCreateMeeting(false, true);
+      });
+
+      test("should create meeting with scalePlan: false and teamPlan: false", async () => {
+        await testCreateMeeting(false, false);
+      });
+    });
+
+    describe("updateMeeting", () => {
+      beforeEach(() => {
+        fetchMocker.mockClear();
+      });
+
+      test.each([
+        [false, false],
+        [false, true],
+        [true, false],
+        [true, true],
+      ])("should update meeting with scalePlan: %s and teamPlan: %s", async (scalePlan, teamPlan) => {
+        await testUpdateMeeting(scalePlan, teamPlan);
+      });
+    });
+  });
```

Comment:
We should do the same with these tests as well.

Reply:
We are working on it, thanks for the feedback